ROTSC/iocBoot/iocROTSC-IOC-01/st.cmd

```diff
@@ -0,0 +1,57 @@
+#!../../bin/windows-x64/ROTSC-IOC-01
+
+## You may have to change ROTSC-IOC-01 to something else
+## everywhere it appears in this file
+
+# Increase this if you get <<TRUNCATED>> or discarded messages warnings in your errlog output
+errlogInit2(65536, 256)
+
+< envPaths
+
+epicsEnvSet "STREAM_PROTOCOL_PATH" "$(ROTSC)/data"
+
+cd "${TOP}"
+
+## Register all support components
+dbLoadDatabase "dbd/ROTSC-IOC-01.dbd"
+ROTSC_IOC_01_registerRecordDeviceDriver pdbbase
+
+##ISIS## Run IOC initialisation 
+< $(IOCSTARTUP)/init.cmd
+
+$(IFDEVSIM) freeIPPort("FREEPORT")  
+$(IFDEVSIM) epicsEnvShow("FREEPORT") 
+$(IFDEVSIM) drvAsynIPPortConfigure("L0", "localhost:57677")
+
+$(IFNOTDEVSIM) drvAsynSerialPortConfigure("L0", "$(PORT)", 0, 0, 0, 0)
+$(IFNOTDEVSIM) asynSetOption("L0", -1, "baud", "$(BAUD=38400)")
+$(IFNOTDEVSIM) asynSetOption("L0", -1, "bits", "$(BITS=8)")
+$(IFNOTDEVSIM) asynSetOption("L0", -1, "parity", "$(PARITY=none)") 
+$(IFNOTDEVSIM) asynSetOption("L0", -1, "stop", "$(STOP=1)")     
+$(IFNOTDEVSIM) asynOctetSetInputEos("L0", -1, "\r")          
+$(IFNOTDEVSIM) asynOctetSetOutputEos("L0", -1, "\r")         
```

Comment:
Use a macro instead of repeating "L0", as in other IOCs (though I expect not all of them)